### PR TITLE
Functional Tests : Audit - BO - Shipping

### PR DESCRIPTION
**Pull request by florine2623**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x
| Description?      | Functional Tests : Audit - BO - Shipping
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | :dolphin: : https://github.com/florine2623/testing_pr/actions/runs/12253340576 ✅ <br/>:seal: : https://github.com/florine2623/testing_pr/actions/runs/12253344811 ✅ 
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: tests/UI/campaigns/audit/BO/09_shipping.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    await boCarriersPage.closeSfToolBar(page);
+
+    const pageTitle = await boCarriersPage.getPageTitle(page);
+    expect(pageTitle).to.contains(boCarriersPage.pageTitle);
+
+    const jsErrors = utilsPlaywright.getJsErrors();
+    expect(jsErrors.length).to.equals(0);
+  });
+
+  it('should go to add new carrier page', async function () {
```
Comment (Progi1984) [inline tests/UI/campaigns/audit/BO/09_shipping.ts]:
```suggestion
  it('should go to \'Shipping > Carriers > New Carrier\' page', async function () {
```

File: tests/UI/campaigns/audit/BO/09_shipping.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    expect(pageTitle).to.contains(boCarriersPage.pageTitle);
+
+    const jsErrors = utilsPlaywright.getJsErrors();
+    expect(jsErrors.length).to.equals(0);
+  });
+
+  it('should edit first line', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToEditCarrierFreeShipping', baseContext);
+
+    await boCarriersPage.goToEditCarrierPage(page, 1);
```
Comment (Progi1984) [inline tests/UI/campaigns/audit/BO/09_shipping.ts]:
```suggestion
    await boCarriersCreatePage.goToPreviousPage(page);
    await boCarriersPage.goToEditCarrierPage(page, 1);
```

File: tests/UI/campaigns/audit/BO/09_shipping.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    await boCarriersPage.goToAddNewCarrierPage(page);
+
+    const pageTitle = await boCarriersCreatePage.getPageTitle(page);
+    expect(pageTitle).to.contains(boCarriersCreatePage.pageTitleCreate);
+
+    const jsErrors = utilsPlaywright.getJsErrors();
+    expect(jsErrors.length).to.equals(0);
+  });
+
+  it('should return to \'Shipping > Carriers\' page', async function () {
```
Comment (Progi1984) [inline tests/UI/campaigns/audit/BO/09_shipping.ts]:
```suggestion
  it('should go to \'Shipping > Carriers > Edit Carrier\' page', async function () {
```

**Review by Progi1984 — COMMENTED** [review]
Some comments

**Review by Progi1984 — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Pull request merged**